secret_number.js

```diff
@@ -1,4 +1,13 @@
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+
+  var secretNumberGenerator = function() {
```

Comment:
Instead of having this function, you can just put the `Math.floor((Math.random()*1000)+1)` directly on line 8 below.